fastkml/kml.py

```diff
@@ -1990,7 +1990,7 @@ def heading(self):
 
     @heading.setter
     def heading(self, value):
-        if isinstance(value, (str, int, float)) and (0 <= float(value) <= 360):
+        if isinstance(value, (str, int, float)) and (-180 <= float(value) <= 180):
```

Comment:
Documentation states that 
```
Values range from 0 to 360 degrees.
```
but my sample file (created by Google Earth) has `-100` as value, so I'm guessing it's actually ±180

Reply:
Not sure if this should be checked after all: 361deg == 1deg

Reply:
ok, just not sure if clients know how to handle 361deg, but I can test it